geometry/src/main/java/coppercore/FieldFinder.java

```diff
@@ -0,0 +1,17 @@
+package coppercore.geometry;
+
+public class FieldFinder {
+    public ConvexPolygon polygon;
+
+    public FieldFinder(ConvexPolygon polygon) {
+        this.polygon = polygon;
+    }
+
+    public void setShape(ConvexPolygon polygon) {
+        this.polygon = polygon;
+    }
+
+    public boolean isRobotInShape(Point p) {
+        return this.polygon.containsPoint(p);
+    }
+}
```

Comment:
Do we want to make a FieldFinder be able to contain more than one polygon or do you think that would be unnecessary?